docs/anchoring-assets/anchor-platform/sep24-user-guide.mdx

```diff
@@ -401,6 +415,489 @@ docker compose up
 
 You should see the logs reporting a successful connection to the postgres database.
 
+## Providing Updates To the Anchor Platform
+
+When the user has provided you the information necessary to facilitate the transaction, you need to update the status of the transaction and provide some of the information you collected to the Anchor Platform.
```

Comment:
If we want to keep it here we should generalize it a bit, since this sentence sounds like we're specifically talking about KYC collection, but the intro to this section should talk about the need to update the platform generally.